### ISSUE TITLE
Labels not associated with any field should be rendered as text fields

### ISSUE THREAD

**Issue by maxxxxxdlp** [specify/specify7#1796]
Labels that are not associated with any fields are used for providing descriptions and usage notes.

These should be rendered as `<p>` rather than `<label>` for accessibility reasons

**Comment by maxxxxxdlp** [comment]
To test this:
In the form definition add a cell with `type="label"`, any string value for the `label` attribute, and no `labelFor` attribute.

Open the resulting form and check via DevTools, that the cell renders a `<p`> element, rather than a `<label>`

**Comment by grantfitzsimmons** [comment]
Works as intended!
